templates/web/common/site_layout.tt.html

```diff
@@ -249,21 +249,9 @@ <h1 itemprop="name">[% title %]
 									[% content %]
 								</div>
 							[% ELSE %]
-								<div class="row">
-									<div class="small-12 column v-space-short">
-										<h1>[% title %]</h1>
-									</div>
-								</div>
 								[% content %]
 							[% END %]
-						[% ELSE %]
```

Comment:
This part should not be removed, otherwise it makes all content from pages that are not product pages, list of products and product edit form have not content.

e.g. the "Discover" page: 
![image](https://user-images.githubusercontent.com/8158668/213659044-babbd168-b23b-4fa2-bbce-ff05f6375cc1.png)